pkg/server/sandbox_remove.go

```diff
@@ -39,9 +41,9 @@ func (c *criContainerdService) RemovePodSandbox(ctx context.Context, r *runtime.
 
 	sandbox, err := c.getSandbox(r.GetPodSandboxId())
 	if err != nil {
-		return nil, fmt.Errorf("failed to find sandbox %q: %v", r.GetPodSandboxId(), err)
-	}
-	if sandbox == nil {
+		if !metadata.IsNotExistError(err) {
```

Comment:
When will the metadata store return an error that is not "NotFound" for a Get request? The "failed to find sandbox" error message below seem to suggest it's the NotFound error anyway...